--- src/PatternLab/Builder.php	2017-06-22 12:02:11.000000000 -0500
+++ src/PatternLab/Builder-new.php	2017-06-22 12:03:09.000000000 -0500
@@ -216,11 +216,18 @@
 				
 				$path          = $patternStoreData["pathDash"];
 				$pathName      = (isset($patternStoreData["pseudo"])) ? $patternStoreData["pathOrig"] : $patternStoreData["pathName"];
-				
+				$pathParts = explode("/",$pathName);
+				array_pop($pathParts);
+				$scssPathName  = implode("/",$pathParts)."/_".$patternStoreData["name"];
+
 				// modify the pattern mark-up
 				$markup        = $patternStoreData["code"];
 				$markupFull    = $patternStoreData["header"].$markup.$patternStoreData["footer"];
 				$markupEngine  = file_get_contents($patternSourceDir."/".$pathName.".".$patternExtension);
+				$scss 		   = @file_get_contents($patternSourceDir."/".$scssPathName.".scss");
+				if ($scss === false) {
+					$scss = "No scss available for this pattern.";
+				}
 				
 				// if the pattern directory doesn't exist create it
 				if (!is_dir($patternPublicDir."/".$path)) {
@@ -232,6 +239,7 @@
 				if (!$exportFiles) {
 					file_put_contents($patternPublicDir."/".$path."/".$path.$suffixMarkupOnly.".html",$markup);
 					file_put_contents($patternPublicDir."/".$path."/".$path.$suffixRaw.".".$patternExtension,$markupEngine);
+					file_put_contents($patternPublicDir."/".$path."/".$path.$suffixRaw.".scss",$scss);
 				}
 				
 			}
